src/transformers/training_args.py

```diff
@@ -1039,25 +1040,25 @@ def __post_init__(self):
             self.greater_is_better = self.metric_for_best_model not in ["loss", "eval_loss"]
         if self.run_name is None:
             self.run_name = self.output_dir
+        if self.framework == "pt" and is_torch_available():
```

Comment:
Let's maybe have a very assert when `self.framwork=="pt"` that `is_torch_available()` is `True`, so we can remove all `is_torch_available()`.

Reply:
That was my plan originally, but unfortunately the Flax examples use `TrainingArguments` directly, there is no `FlaxTrainingArguments`! Do you want me to add `FlaxTrainingArguments` (with `framework = "flax"`) and change the Flax examples to use it? Or should we just use this hack for now and split the `TrainingArguments` in a future PR?

Reply:
Ah, hadn't thought of that. Let's leave it as is for now then, but we really need that `BaseTrainingArguments` class ;-)